### PR TITLE
Work and about section 

### PR DESCRIPTION
**Pull request by learnwithalfred**
Four sections have been added and styled using flexbox and CSS grid

Screenshot 2022-04-21 at 03 33 03

### CODE REVIEW

**Review by nicupop729 — CHANGES_REQUESTED** [review]
Hi @learnwithalfred, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
 - [X] Correct GitHub flow practice :heavy_check_mark:
 - [X] Good commit messages :heavy_check_mark:
 - [X] Professional README :heavy_check_mark:
 - [X] Linters are passing :heavy_check_mark:
 - [X] Clean codebase :heavy_check_mark:

## Required Changes ♻️
Because the About section was done in a previous pull request, I will mention it here and not with inline comments:

- [x] index.html - starting with row 43 - the developer experience needs to be a completely separate section. Kindly, end the section with the class 'developer' at row 42 in index.html, and start another one which should hold the experiences, as the project requirements specifically ask for;

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -187,4 +192,176 @@ body {
...
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #edecff;
+  }
```
[the remark this answers is not in the document]
Comment (learnwithalfred) [reply style.css:249]: Thanks, I just did that

File: style.css [hunk 2]
```diff
@@ -187,4 +192,176 @@ body {
...
     color: #e91683;
   }
+
+  .work {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    color: white;
+    position: relative;
+  }
```
[the remark this answers is not in the document]
Comment (learnwithalfred) [reply style.css]: Thank you.
I overlooked that.

I have to pay attention to details

File: style.css [hunk 3]
```diff
@@ -9,6 +9,11 @@ body {
 }
 
 @media only screen and (max-width: 550px) {
```
Comment (topeogunleye) [inline style.css:11]: - [ ] [OPTIONAL] I know this is for Mobile design, but it is advisable to build your website mobile-first. This means style the website for mobiles devices first, then use `min-width` to style the website for desktops or bigger devices.

**Review by topeogunleye — APPROVED** [review]
### Hi @learnwithalfred, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
> 
> ## [Optional] suggestions:
> * [ ]  Please check my inline comments.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (learnwithalfred) [reply style.css:11]: I will add that

Thanks

Comment (topeogunleye) [reply style.css:11]: You're welcome

**Pull request merged**